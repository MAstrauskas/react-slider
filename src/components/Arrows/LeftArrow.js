import React from "react";
import Arrow from "../../assets/icons/left-arrow.svg";

import "./Arrows.css";

const LeftArrow = (props) => {
    return (
        <div className="arrow" onClick={props.handlePrevSlide}>
            <img src={Arrow} alt="Left Arrow" aria-hidden="true" />
        </div>
    )
};

export default LeftArrow;