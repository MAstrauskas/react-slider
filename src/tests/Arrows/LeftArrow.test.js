import React from "react";
import { create } from "react-test-renderer";
import LeftArrow from "../../components/Arrows/LeftArrow";

describe("Left Arrow Component", () => {
    test("it matches the snapshot", () => {
        const component = create(<LeftArrow/>);
        expect(component.toJSON()).toMatchSnapshot();
    })
});

