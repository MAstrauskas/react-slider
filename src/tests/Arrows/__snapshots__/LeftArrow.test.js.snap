// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Left Arrow Component it matches the snapshot 1`] = `
<div
  className="arrow"
>
  <img
    alt="Left Arrow"
    aria-hidden="true"
    src="left-arrow.svg"
  />
</div>
`;
